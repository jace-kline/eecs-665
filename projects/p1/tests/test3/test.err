FATAL [1,52]: Illegal character \
FATAL [1,75]: Illegal character \
