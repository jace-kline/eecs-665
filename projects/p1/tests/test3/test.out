ID:just [1,8]
ID:put [1,13]
ID:a [1,17]
ID:space [1,19]
ID:after [1,25]
ID:an [1,31]
ID:escape [1,34]
ID:character [1,41]
ID:n [1,53]
ID:manually [1,58]
ID:added [1,67]
ID:a [1,73]
ID:n [1,76]
ID:outside [1,78]
ID:of [1,86]
ID:a [1,89]
ID:string [1,91]
ID:literal [1,98]
EOF [1,105]
